plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -248,10 +253,47 @@ export default class ChatComposer extends Component {
       return;
     }
 
+    if (await this.reactingToLastMessage()) {
+      return;
+    }
+
     await this.args.onSendMessage(this.draft);
     this.composer.textarea.refreshHeight();
   }
 
+  async reactingToLastMessage() {
+    // Check if the message is a reaction to the latest message in the channel.
+    const message = this.draft.message.trim();
+    let reactionCode = "";
+    if (message.startsWith("+")) {
+      const reaction = message.substring(1);
+      // First check if the message is +{emoji}
+      if (replacements[reaction]) {
+        reactionCode = replacements[reaction];
+      } else {
+        // Then check if the message is +:{emoji_code}:
+        const emojiCode = reaction.substring(1, reaction.length - 1);
+        if (emojiExists(emojiCode)) {
+          reactionCode = normalizeEmoji(emojiCode);
```

Comment:
```suggestion
        reactionCode = normalizeEmoji(emojiCode);
```

I think this can be simplified to the above.